plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -53,26 +46,7 @@
 public abstract class BaseElasticsearchConnectorTest
         extends BaseConnectorTest
 {
-    private final String image;
-    private ElasticsearchServer elasticsearch;
-    protected RestHighLevelClient client;
-
-    BaseElasticsearchConnectorTest(String image)
-    {
-        this.image = image;
-    }
-
-    @Override
-    protected QueryRunner createQueryRunner()
-            throws Exception
-    {
-        elasticsearch = new ElasticsearchServer(image, ImmutableMap.of());
-
-        HostAndPort address = elasticsearch.getAddress();
-        client = new RestHighLevelClient(RestClient.builder(new HttpHost(address.getHost(), address.getPort())));
-
-        return createElasticsearchQueryRunner(elasticsearch.getAddress(), TpchTable.getTables(), ImmutableMap.of(), ImmutableMap.of(), 3);
-    }
```

Comment:
Why is this moved to subclasses?

Reply:
Abstract test classes shouldn't dictate how the query runner is set up. It's better to be explicit and let each subclass define their own runners. Makes it clear if the test depends on some specific configuration.

That's how we've been doing in other connectors - this one was an outlier.